busted/outputHandlers/json.lua

```diff
@@ -1,25 +1,39 @@
-local json = require 'dkjson'
+local json = require("dkjson")
```

Comment:
Lets not change the coding style please. I don't like this style at all myself and plan to change it, but that will be done in bulk with a deterministic tool, probably `stylua`. In any case it is used consistently throughout the code base and shouldn't be changed incidentally in an unrelated PR.

Reply:
Sure, my bad.  Stylua got out of hand.